### ISSUE TITLE
Update peer dependency to add react ^18.0.0

### ISSUE THREAD

**Issue by anmol5varma** [rnosov/react-reveal#155]
Update peerDependency to `react@"^16.8.0 || ^17.0.0-rc.1 || ^18.0.0"` in package.json

**Comment by Nautman** [comment]
I agree that it would be really good! But it doesn't seem as though this package is updated anymore?

**Comment by Mutesa-Cedric** [comment]
please use [react-swift-reveal](https://www.npmjs.com/package/react-swift-reveal), it has the same API surface as react-reveal and its currently being maintained.
